platforms/openid-connect/README.md

```diff
@@ -0,0 +1,110 @@
+## Unleash Community Docker Image with OpenID Connect Support
+
+**Useful links:**
+
+- [Docker image on dockerhub](https://hub.docker.com/r/unleashorg/unleash-community-openid-connect)
+- [passport-openid-connect](https://www.passportjs.org/packages/passport-openidconnect/)
+
+For an example usage see the example below.
+You can also visit the [passport](https://www.passportjs.org/docs/) and [openid-connect](https://www.passportjs.org/packages/passport-openidconnect/) documentations for further information.
+
+## Example
+
+This shows the usage of the OpenIdConnectStrategy with a keycloak realm as IDP.
+It authenticates the user and checks some custom roles to give the user some initial permissions. 
+It also updates the role in case it changed, so you can completely control unleash permissions through roles in keycloak.
+You need to configure your client accordingly, so it returns roles as a `resource_access` object.
+
+If you only need the authentication, just remove that part.
+
+```javascript
+const unleash = require('unleash-server')
+
+function openIdConnect(app, config, services) {
+    const {userService} = services;
+
+    passport.use(new OpenIdConnectStrategy({
+        issuer: `${host}/realms/${realm}`,
+        authorizationURL: `${host}/realms/${realm}/protocol/openid-connect/auth`,
+        tokenURL: `${host}/realms/${realm}/protocol/openid-connect/token`,
+        userInfoURL: `${host}/realms/${realm}/protocol/openid-connect/userinfo`,
+        clientID: clientID,
+        clientSecret: '',
+        callbackURL: `${contextPath}/api/auth/callback`,
+        skipUserProfile: false
+    }, async (issuer, uiProfile, idProfile, context, idToken, accessToken, refreshToken, params, cb) => {
+
+        let raw = JSON.parse(uiProfile._raw)
+        const roles = raw.resource_access[clientID].roles;
+
+        const email = idProfile.emails[0].value
+
+        let role = RoleName.VIEWER;
```

Comment:
For the example we could also strip out the roles functionality as it is very environment and keycloak specific?

This could be helpful to someone, but I am not sure this is the place to document this 😁

@chriswk @ivarconr, anyone has an opinion on this?

Reply:
I'll just remove it for now. Its not basic example as you're already noticed